components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/clients/dto/ApiCreatorData.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.apimgt.impl.clients.dto;
+
+/**
+ * Maps the response relevant to the Rudder's /api/v1/choreo/mappings/namespace-to-project/{namespace} endpoint
+ */
+public class ApiCreatorData {
+    public String organization_id;
+    String project_id;
```

Comment:
This is a wrong way of setting fields to a DTO. 
We should have private variable names and public getters and setters. I understand that we wanted to address this "_" in the name, the proper way of doing this is to use best practices with variable and method names and use annotations to set the actual response field mapping. Its fine to skip this for now but make sure this is fixed in a different PR later